Sprint-1/interpret/percentage-change.js

```diff
@@ -12,11 +12,22 @@ console.log(`The percentage change is ${percentageChange}`);
 // Read the code and then answer the questions below
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
+// There are 3 function calls
```

Comment:
Is `Number()` a function call?

Reply:
Yes from what I gathered: 
The Number() function is a built-in JavaScript function that converts a given value to a number. If the conversion is successful, it returns a numeric value; if it fails (e.g., the input is not a valid number), it returns NaN (Not-a-Number).